src/components/CommentCard.jsx

```diff
@@ -0,0 +1,15 @@
+import {ArrowUpOutlined,ArrowDownOutlined } from '@ant-design/icons'
+import { incrementCommentVoteCount } from './utils/api'
+function CommentCard({comment,setUpdateComment,updateComment}) {
+    function changeVotes({comment_id},vote) {
+        incrementCommentVoteCount(comment_id,vote).then((comment)=>{setUpdateComment(updateComment+1)})
```

Comment:
Whenever you vote on comments, the order changes weirdly, seemingly not following a pattern. That isn't great and extremely confusing to the user - I think you'd want to pass a callback to this setState and then use a copy of the current state, because it causes rendering issues otherwise.